storybookjs/storybook
# CLI: Add automigration for GFM in MDX

Closes https://github.com/storybookjs/storybook/issues/20091

## What I did

I've added an automigration that adds GFM support to addon-docs via adding an addon
I've created the addon that adds GFM config

The addon is a tool to assist users migrate, it's not supposed to be used indefinitely.

The reaosn we did it this way, is because it made writing the auto-migration much much much easier. User will have to do a manual step, but it should get them into a working state

## How to test

After running the automigration you should get an extra addon in your addon array
You should also heb the addon added to your package.json
Your storybook should now support GFM without having to configure addon-docs.

There should be a deprecation notice in the command-loine telling you (once again) that you should perform the manual migration.
